plugins/flytekit-pydantic/setup.py

```diff
@@ -4,7 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=1.7.0b0", "pydantic"]
+plugin_requires = ["flytekit>=1.7.0b0", "pydantic<2"]
```

Comment:
Can we also leave a warning in the [README.md](https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-pydantic/README.md) explaining that we're deprecating this plugin?

Reply:
@eapolinario 
> how is this going to work though? Do we also do a version check of the backend?
No this is just for supporting the case I've mentioned above.
we didn't support this before and I think we should do it.